www/src/config.ts

```diff
@@ -42,10 +42,15 @@ export const ALGOLIA = {
   apiKey: "892c4647b96fe1b3d0b7d8de1c5b5e40",
 };
 
-export type Sidebar = Record<
-  KnownLanguageCode,
-  Record<string, { text: string; link: string }[]>
->;
+type OuterHeaders = "Create T3 App" | "Usage" | "Deployment";
```

Comment:
use this type here too:

https://github.com/t3-oss/create-t3-app/blob/1f4dac9527fbf44009e1fecc64e248a748073e7b/www/src/config.ts#L93

Reply:
done